src/app/(guest)/profile/[token]/page.tsx

```diff
@@ -1,19 +1,21 @@
+import PreviousExamsTable from '@/components/PreviousExamsTable';
 import { BASE_URL } from 'src/services/api';
+import User from 'src/types/User';
 
 interface ProfileProps {
   params: {
     token: string;
   };
 }
-// @ts-expect-error Server Component
-const profile: React.FC<ProfileProps> = async ({ params }) => {
+
+export default async function profile({ params }: ProfileProps) {
```

Comment:
Change the typing of the variable to `const profile: React.FC<ProfileProps>` to be consistent with the remaining project.

You will need to add `// @ts-expect-error Server Component` in the line on top of the const declaration because of the async signature